paderbox/transform/module_resample.py

```diff
@@ -50,11 +50,14 @@ def resample_sox(signal: np.ndarray, *, in_rate, out_rate):
         signal: Signal as one-dimensional np.ndarray: Shape (T,)
         in_rate: Probably as an integer
         out_rate: Probably as an integer
+        normalize: If True, normalize input to be between -1 and 1, befor
+            calling sox. After resampling, the normalization is reverted.
+            Sox has the issue, to clip some values, if they are above 1.
```

Comment:
This paragraph has a few typos, maybe write:

If True, the input is normalized to the range between -1 and 1 before calling sox. The normalization is reverted after resampling. Sox clips some values if they exceed 1.

Reply:
Done